### ISSUE TITLE
[release 1.0.1] TypeError: Cannot read property 'reason' of undefined

### ISSUE THREAD

**Issue by benoitberanger** [bids-standard/legacy-validator#603]
Hello,

I have a problem with the CLI version, but only with the current release.
With the current release [1.0.1](https://github.com/bids-standard/bids-validator/releases/tag/1.0.1) I have the error : 
```
(node:4340) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
(node:4340) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'reason' of undefined
    at headerFields (/home/benoit.beranger/bids-validator/validators/headerFields.js:41:64)
    at NIFTI.validate.then.niftiIssues (/home/benoit.beranger/bids-validator/validators/bids/fullTest.js:151:54)
    at process._tickCallback (internal/process/next_tick.js:68:7)
(node:4340) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:4340) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```
But **no problem** with the previous release [1.0.0](https://github.com/bids-standard/bids-validator/releases/tag/1.0.0), neither the previous one [0.27.10](https://github.com/bids-standard/bids-validator/releases/tag/0.27.10)

Ben

**Comment by benoitberanger** [comment]
By the way, I always have this line, so I don't think it's relevant :
`[DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.`

**Comment by nellh** [comment]
Thanks for the report!

Few things I'm seeing for us to fix here:
- [x] We should fix that Buffer usage, it is just a warning but future versions of Node will remove this usage.
- [x] It looks like we're lacking test coverage for issue 39 - "Not all subjects/sessions/runs have the same scanning parameters."
- [x] When issue 39 is found it throws this error as of 1.0.1

**Comment by nellh** [comment]
This will be fixed in 1.0.2

**Comment by benoitberanger** [comment]
Great, thank you !
